test/citestutils.js

```diff
@@ -112,7 +112,7 @@ function testBackup(params, databaseName, outputStream, callback) {
         }
       });
       tail.on('error', function(err) {
-        callback(err);
+        return Promise.reject(err);
```

Comment:
If this error is encountered we'll get a "Unhandled promise rejection" error in Node because this `on` event listener is separate from all the promise chains.
We could probably set up a new promise that rejects for the `error` listener and resolves for the `close` event and add it to the `promises` array.
That said I don't think this is very likely anyway, we could probably just remove this `error` listener and just let Node surface an unhandled error event if it happens (which newer Node versions do anyway IIRC).